lms/static/styles/components/_ErrorDisplay.scss

```diff
@@ -12,7 +12,6 @@
 
   &__links {
     white-space: normal;
-    line-height: 1.25em;
```

Comment:
It was a change done originally to make the word wrapping of the links look better because the default line-height is fixed at `px` rather than `em`, but now since the font is back to default size, things look okay when wrapped. I just decided to remove the override.